html/scripts/htmlLoad.js

```diff
@@ -0,0 +1,129 @@
+// ----------------Javascript begins-----------------------------
+
+function uuidv4() {
+  return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
+    (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
+  );
+}
+
+// let inputText;
+// let setNewId = 0;
+let tasks = [
+  { id: uuidv4(), isCompleted: false, isEditing: false, content: 'cook barbeque' },
+  { id: uuidv4(), isCompleted: true, isEditing: false, content: 'cook WOK' },
+  { id: uuidv4(), isCompleted: false, isEditing: false, content: 'cycling' },
+];
+
+function deleteTask(id) {
+  tasks = tasks.map(task => {
+    if (id === task.id) return null;
+    return task;
+  }).filter(Boolean);
+
+  render();
+}
+
+function editTask(id) {
+  tasks = tasks.map(task => {
+    if (id === task.id) return {...task, isEditing: true};
+    return task;
+  }).filter(Boolean);
+  
+  render();
+}
+
+function formOnBlur(id) {
+  tasks = tasks.map(task => {
+    if (id === task.id) return {...task, isEditing: false};
+    return task;
+  })
+
+  render();
+}
+
+function saveEditedTask(id) {
+  const editedContent = document.querySelector('.text-edit').value;
+  tasks = tasks.map(task => {
+    if (id === task.id) return {...task, isEditing: false, content: editedContent};
+    return task;
+  })
+
+  render();
+}
+
+function editableRow({id, content, isCompleted, isEditing}) {
+  if (isEditing) return `<input onblur="formOnBlur('${id}')" type="text" class="text-edit" value="${content}">
+  <button onclick="saveEditedTask('${id}')" class="save-button" type="button">💾</button>`;
+  
+  return `${isCompleted === true ? `<s>${content}</s>` : `<span onclick="editTask('${id}')">${content}</span>`}`
+}
+
+
+function renderListRow(task) {
+  const { id, isCompleted } = task;
+  return `
+    <li>
+      <input onclick="changeState('${id}')" class="task-state" type="checkbox" ${isCompleted === true ? 'checked' : ''}>
```

Comment:
```suggestion
      <input onclick="changeState('${id}')" class="task-state" type="checkbox" ${isCompleted ? 'checked' : ''}>
```

Think why it gonna work and replace in all other places